.github/workflows/release.yml

```diff
@@ -13,11 +15,49 @@ jobs:
         os: [windows-latest]
 
     steps:
+      - name: Get Numerical Version Number from Tag
+        if: startsWith(github.ref, 'refs/tags')
+        shell: bash
+        id: versionNumber
+        run: echo ::set-output name=tag::${GITHUB_REF/refs\/tags\/v/}
+
       - name: Checkout
         uses: actions/checkout@v2.0.0
         with:
           lfs: true
 
+      - name: Cache Choco
+        id: cache-choco
+        uses: actions/cache@v2
+        with:
+          path: |
+            C:\ProgramData\chocolatey\lib\xmlstarlet.portable
```

Comment:
Is it guaranteed that the path will always be the windows format?

Reply:
Yes, until/unless we add mac support in which case this function will not work and I will have to find an alternative.